src/CultureInfoGenerator/Program.cs

```diff
@@ -58,24 +59,7 @@ public static partial class CultureInfoCache
                     | CultureTypes.InstalledWin32Cultures
                     | CultureTypes.SpecificCultures);
 
-                var format = "            \"{0}\"";
-
-                for (int i = 0; i < cultures.Length; i++)
-                {
-                    var culture = cultures[i];
-
-                    writer.Write(format, culture.Name);
-
-                    if (i < cultures.Length - 1)
-                    {
-                        writer.WriteLine(",");
-                    }
-                    else
-                    {
-                        // Last entry
-                        writer.WriteLine();
-                    }
-                }
+                writer.WriteLine(string.Join(",\n", cultures.Select(c => $"            \"{c.Name}\"")));
```

Comment:
I think it's reasonable to argue that using `string.Join` is clearer than a `for()` loop.